### PR TITLE
fixed portal view

### PR DESCRIPTION
**Pull request by tiluong**

### CODE REVIEW

File: src/scss/components/application/_application.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@
   width: $width;
 }
 .application {
+  overflow: hidden;
```
Comment (ejmockler) [inline src/scss/components/application/_application.scss:22]: It seems this doesn't fix the x-overflow unfortunately, there's side scrolling on smaller screen sizes :/

**Review by ejmockler — COMMENTED** [review]
the application kinda looks cluttered with the footer down there anyways, I'd say we're chillin on the portal as-is!

File: src/scss/components/application/_application.scss (hunk 1 above)
Comment (ejmockler) [reply src/scss/components/application/_application.scss:22]: the width unit for the footer should scale with the window/screen width, like `vw`

**Comment by Kyle1668** [comment]
I'm getting this 
image

**Pull request closed without merging**